app/controllers/application_controller.rb

```diff
@@ -1,8 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
-  
+
+
   rescue_from CanCan::AccessDenied do |exception|
     flash[:error] = "Access Denied"
     redirect_to root_url
   end
+
```

Comment:
Sure. I think I was trying something out, but then removed it. I can delete that.